scripts/src/core/bump-version-deps.ts

```diff
@@ -0,0 +1,20 @@
+import { readdirSync } from 'fs';
+import { PATHS, getPackageJson } from '../config/paths';
+import { writeFile2 } from '../utils/file-system.utils';
+import { resolve } from 'path';
+
+(async () => {
+  const rootPackageJson = getPackageJson('root');
+  const version = rootPackageJson.version;
+
+  const folders = readdirSync(PATHS.packages);
+  for (const folder of folders) {
+    const packageJSON = getPackageJson('dist', folder);
+    Object.keys(packageJSON.peerDependencies).forEach((key) => {
+      if (key.includes('@igo2')) {
+        packageJSON.peerDependencies[key] = `^${version}`;
```

Comment:
Since we use the  workspace feature to add a cross-reference between our packages, we need to overwrite it with the version number deployed.